app/mailers/notification_mailer.rb

```diff
@@ -0,0 +1,10 @@
+# Send admin notifications
+class NotificationMailer < ApplicationMailer
+  def submission(soundbite:)
+    @soundbite = soundbite
+    mail(
+      to: 'rmacg89+ukip@gmail.com',
+      subject: 'New Submission'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.